python/lsst/meas/deblender/deblend.py

```diff
@@ -467,67 +470,67 @@ class MultibandDeblendConfig(pexConfig.Config):
-                                        "This field is ignored if fitPositions is False."))
-    minTranslation = pexConfig.Field(dtype=float, default=1e-8,
+
+    # Blend Configuration options
+    minTranslation = pexConfig.Field(dtype=float, default=1e-3,
                                      doc=("A peak must be updated by at least 'minTranslation' (pixels)"
                                           "or no update is performed."
                                           "This field is ignored if fitPositions is False."))
-    maxTranslation = pexConfig.Field(dtype=float, default=2,
-                                     doc=("A peak must not be updated by more than 'maxTranslation' pixels."
-                                          "This field is ignored if fitPositions is False."))
-    requireTranslation = pexConfig.Field(dtype=int, default=10,
-                                       doc=("If fitPositions is True, the first 'requireTranslations'"
-                                            "iterations will fit the peaks, then positions are"
-                                            "updated on every 'skipTranslation' iterations."))
-    skipTranslation = pexConfig.Field(dtype=int, default=10,
-                                      doc=("If fitPositions is True, the first 'requireTranslations'"
-                                           "iterations will fit the peaks, then positions are"
-                                           "updated on every 'skipTranslation' iterations."))
+    refinementSkip = pexConfig.Field(dtype=int, default=10,
+                                      doc=("If fitPositions is True, the positions and box sizes are"
+                                           "updated on every 'refinementSkip' iterations."))
     translationMethod = pexConfig.Field(dtype=str, default="default",
                                         doc=("Method to use for fitting translations."
                                              "Currently 'default' is the only available option,"
                                              "which performs a linear fit, but it is possible that we"
                                              "will use galsim or some other method as a future option"))
+    edgeFluxThresh = pexConfig.Field(dtype=float, default=1.0,
+                                     doc=("Boxes are resized when the flux at an edge is "
+                                          "> edgeFluxThresh * bg_rms"))
+    exactLipschitz = pexConfig.Field(dtype=bool, default=False,
+                                     doc=("Calculate exact Lipschitz constant in every step"
+                                          "(exact_lipschitz is True) or only calculate the"
+                                          "Lipschitz constant with significant changes in A,S"
+                                          "(exact_lipschitz is False)"))
```

Comment:
Just say `True` or `False` and leave out the `exact_lipschitz` part (which is confusing because of the name change).